src/microbe_stage/MicrobeInternalCalculations.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.Collections.Generic;
+
+public static class MicrobeInternalCalculations
+{
+    public static float CalculateSpeed(IEnumerable<OrganelleTemplate> organelles, MembraneType membraneType,
+        float membraneRigidity)
+    {
+        float microbeMass = Constants.MICROBE_BASE_MASS;
+
+        float organelleMovementForce = 0;
+
+        foreach (var organelle in organelles)
+        {
+            microbeMass += organelle.Definition.Mass;
+
+            if (organelle.Definition.HasComponentFactory<MovementComponentFactory>())
+            {
+                if (organelle.Orientation == 3)
+                {
+                    organelleMovementForce += Constants.FLAGELLA_BASE_FORCE
+                        * organelle.Definition.Components.Movement.Momentum / 100.0f;
```

Comment:
Why divide it by 100.0f?

Reply:
inherited code. I assume that we can take that off and just make the constant smaller, but that's going to increase testing scope on what's already a big PR.